js/apps/account-ui/src/resources/ResourcesTab.tsx

```diff
@@ -174,7 +185,7 @@ export const ResourcesTab = () => {
                 </a>
               </Td>
               <Td dataLabel={t("permissionRequests")}>
-                {resource.shareRequests.length > 0 && (
+                {(resource.shareRequests?.length || 0) > 0 && (
```

Comment:
I'd prefer this condition terminates, rather than falling back. But this is a personal preference.
```suggestion
                {resource.shareRequests && resource.shareRequests.length > 0 && (
```